WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/search_results/wca-regulations.json")).freeze
```

Comment:
Yes, as long as the website and the Regulations stays as independent as they are right now it's fine.